### PR TITLE
various

### PR DESCRIPTION
**Pull request by stefanfoulis**
sorry for the combined pull-request... in a hurry :-)

see individual commit messages for details

### CODE REVIEW

**Comment by GaretJax** [comment]
LGTM

**Comment by Chive** [comment]
:+1:

File: aldryn_config.py [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 
+SYSTEM_FIELD_WARNING = 'WARNING: this field is auto-written. Please do not change it here.'
+
 
 class Form(forms.BaseForm):
-    languages = forms.CharField('Languages', required=True, initial='["en", "de"]')
+    languages = forms.CharField(
+        'Languages',
+        required=True,
+        initial='["en", "de"]',
```
Comment (intellectronica) [inline aldryn_config.py:14]: Why do we include `de` as a default?

**Comment by stefanfoulis** [comment]
tnx

**Pull request merged**